frontend/src/app/shared/components/chip/chip.component.scss

```diff
@@ -0,0 +1,95 @@
+@import 'colors';
+@import 'variables';
+
+.chip {
+  border: 2px solid $primary-color-050;
+  color: $primary-color-010;
+  background-color: transparent;
+  padding: 4px 8px;
+  display: inline-flex;
+  border-radius: $border-radius-md1;
```

Comment:
Incearca sa gasesti o ordine in suflet. Eu si ... Petrut mai nou cand scriem proprietati le gasim locul in ordinea asta
```

position
display
flex
width
height

margin 
padding

font-size: 
font-weight:
color

background-color
background-size
bacground-image

border
border-radius
```

deci pentru tine as face o sugestie de genul:

```
display: inline-flex;
padding: 4px 8px;

color: $primary-color-010;
background-color: transparent;
border: 2px solid $primary-color-050;
border-radius: $border-radius-md1;
```

Poti sa ma contrazici dar e mai citet, mai usor de inteles codul, iar daca fiecare block de css respecta regula asta, il citesti / identifici mai rapid

Reply:
fixed